scipy/stats/tests/test_stats.py

```diff
@@ -7439,7 +7439,8 @@ def test_inf_weight(self):
 
 
 class TestWassersteinDistance:
```

Comment:
Please create a new class `TestWassersteinDistanceND` and add the new tests there instead of interspersing the two.

Reply:
Done!